src/gen/libcuopt.jl

```diff
@@ -1,21 +1,3 @@
-# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
-# SPDX-License-Identifier: Apache-2.0
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# Disable JuliaFormatter for this file.
-#!format: off
-
```

Comment:
This header needs to be re-instated. I had `!!format: off` so that formatting didn't format this file. It'd make it easier to tell what the differences were for the new API.

Reply:
I have generated it back and added format off so the comparison is better.